Code/Tools/TestImpactFramework/CMakeLists.txt

```diff
@@ -14,11 +14,18 @@ if(PAL_TRAIT_TEST_IMPACT_FRAMEWORK_SUPPORTED)
     add_subdirectory(Runtime)
     add_subdirectory(Frontend)
     ly_add_pytest(
-        NAME TiafToolsTest
-        PATH ${LY_ROOT_FOLDER}/scripts/build/TestImpactAnalysis/test_tiaf_tools.py
+        NAME TIAFToolsTest
```

Comment:
nit: This naming convention is something in the coding standards, It is preferred to use camel case even for acronyms, so this should remain `TiafToolsTest`